src/components/swap/TradePrice.tsx

```diff
@@ -58,7 +58,7 @@ export default function TradePrice({ price, showInverted, setShowInverted }: Tra
       }}
       title={text}
     >
-      <Text fontWeight={500} color={theme.textPrimary}>
+      <Text fontWeight={500} fontSize={14} color={theme.textPrimary}>
```

Comment:
I'm wondering if ThemedText.BodySmall can be used here, although ThemedText.BodySmall does have a fontWeight of 400... is there a jira or link to the original figma? if 500 is needed this looks fine to me :P

Reply:
Oooh i just checked the ticket and checked on my local version of the app, the text that Cal wants you to match does have fontWeight 400 so I think this would be a good place to use ThemedText.BodySmall!

Reply:
thanks!